### PR TITLE
New product VirusTotal

### PR DESCRIPTION
**Pull request by vutut**
**Type of pull request:** product addition

**Related issues:** n/a

---

Adding a new assessment for VirusTotal.

### CODE REVIEW

File: products/virustotal.toml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+]
+
+[rubric.list-collected]
+value = "generally"
+citations = [
+  "We collect information, including personal information, from visitors to the Site, as well as our registered users, customers and partners. We also collect certain information when anyone uses the Site. We collect information about the use of the Site and interaction with the Service, and information extracted from any information, text, graphics, URLs, files, audio, video, photos or other materials uploaded, downloaded or appearing on the Service (\"Samples\") or submitted through the Service."
+]
+notes = "The policy includes a specific listing of data collected, but with use of language like \"such as\" which allows for collection of other data."
```
Comment (milesmcc) [inline products/virustotal.toml:79]: Small thing: this needs to be an array, not a string. Thanks!

Comment (vutut) [reply products/virustotal.toml]: Fixed, noticed when checking after PR submission.

**Comment by vutut** [comment]
All fixed per review (I think), thank you @ibarakaiev!

**Comment by ibarakaiev** [comment]
Looks good me! Thanks for doing this.

**Pull request merged**